sale_partner_incoterm/models/sale_order.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Opener B.V. (<https://opener.am>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    incoterm_address_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Incoterm Address",
+    )
+
+    @api.onchange("partner_id")
+    def onchange_partner_id_set_incoterm(self):
```

Comment:
@StefanRijnhart As Odoo changed sale orders to full computed fields, shouldn't we use them too (store=True, readonly=False) ?